### PR TITLE
lr/ovs-slaves: expose slaves on bus/libnm

### PR DESCRIPTION
**Pull request by lkundrak**
The same is done for bonds/bridges/teams.

### CODE REVIEW

File: src/devices/ovs/nm-device-ovs-bridge.c [hunk 1]
```diff
@@ -134,6 +134,9 @@ nm_device_ovs_bridge_init (NMDeviceOvsBridge *self)
 static const NMDBusInterfaceInfoExtended interface_info_device_ovs_bridge = {
 	.parent = NM_DEFINE_GDBUS_INTERFACE_INFO_INIT (
 		NM_DBUS_INTERFACE_DEVICE_OVS_BRIDGE,
+		.properties = NM_DEFINE_GDBUS_PROPERTY_INFOS (
+			NM_DEFINE_DBUS_PROPERTY_INFO_EXTENDED_READABLE_L ("Slaves", "ao", NM_DEVICE_SLAVES),
```
Comment (thom311) [inline src/devices/ovs/nm-device-ovs-bridge.c]: new code should not use `NM_DEFINE_DBUS_PROPERTY_INFO_EXTENDED_READABLE_L`. Instead, use `NM_DEFINE_DBUS_PROPERTY_INFO_EXTENDED_READABLE`. The "L" stands for "legacy".

File: libnm/nm-device-ovs-bridge.c [hunk 2]
```diff
@@ -26,12 +26,21 @@
...
+
+	LAST_PROP
+};
 
 /**
  * NMDeviceOvsBridge:
  */
 struct _NMDeviceOvsBridge {
 	NMDevice parent;
+	GPtrArray *slaves;
```
Comment (thom311) [inline libnm/nm-device-ovs-bridge.c]: slaves leaks. In both device types.

File: introspection/org.freedesktop.NetworkManager.Device.OvsBridge.xml [hunk 3]
```diff
@@ -7,6 +7,16 @@
   -->
   <interface name="org.freedesktop.NetworkManager.Device.OvsBridge">
 
+    <!--
+        Slaves:
+
+        Array of object paths representing ports which are currently slaved to
+        this bridge.
```
Comment (bengal) [inline introspection/org.freedesktop.NetworkManager.Device.OvsBridge.xml]: slaved -> enslaved?

File: introspection/org.freedesktop.NetworkManager.Device.OvsPort.xml [hunk 4]
```diff
@@ -7,6 +7,16 @@
   -->
   <interface name="org.freedesktop.NetworkManager.Device.OvsPort">
 
+    <!--
+        Slaves:
+
+        Array of object paths representing interfaces which are currently slaved to
+        this port.
```
Comment (bengal) [inline introspection/org.freedesktop.NetworkManager.Device.OvsPort.xml]: The same. (I know, other device types use 'slaved' as well, but it sounds wrong to me).

File: libnm/nm-device-ovs-bridge.c (hunk 2 above)
Comment (lkundrak) [reply libnm/nm-device-ovs-bridge.c]: Are you sure? They're cleared in dispose().

File: introspection/org.freedesktop.NetworkManager.Device.OvsBridge.xml (hunk 3 above)
Comment (lkundrak) [reply introspection/org.freedesktop.NetworkManager.Device.OvsBridge.xml]: Yes. I've corrected it in some places while cargo-culting, lazy to fix the originals, and left some oversights. Added a commit that fixes the rest now.

**Pull request merged**